tests/antithesis/avalanchego/main.go

```diff
@@ -614,22 +570,10 @@ func (w *workload) confirmPChainTx(ctx context.Context, tx *ptxs.Tx) {
 		s, err := client.AwaitTxDecided(ctx, txID, 100*time.Millisecond)
 		if err != nil {
 			log.Printf("failed to determine the status of a P-chain transaction %s on %s: %s", txID, uri, err)
-			assert.Unreachable("failed to determine the status of a P-chain transaction", map[string]any{
-				"worker": w.id,
-				"txID":   txID,
-				"uri":    uri,
-				"err":    err,
-			})
 			return
 		}
 		if s.Status != status.Committed {
 			log.Printf("failed to confirm P-chain transaction %s on %s: status == %s", txID, uri, s.Status)
-			assert.Unreachable("failed to confirm a P-chain transaction", map[string]any{
-				"worker": w.id,
-				"txID":   txID,
-				"uri":    uri,
-				"status": s.Status,
-			})
```

Comment:
It's my understanding that this assertion could remain. (I'd be interested if it couldn't stay)